pkg/cmd/pulumi/logout.go

```diff
@@ -74,6 +74,7 @@ func newLogoutCmd() *cobra.Command {
 			var be backend.Backend
 			var err error
 			if filestate.IsFileStateBackendURL(cloudURL) {
+				fmt.Printf("Logged out of %s\n", cloudURL)
```

Comment:
Didn’t notice this at the time, but informational messages like this really ought to go to stderr.

Reply:
Semi-related, but I was talking with @susanev around what the CLI pipe experience might be like since every change to output could be a breaking change from that perspective.